From 0f650304787c7f4f3c527368c77bef1d352f636e Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Fri, 2 Apr 2021 12:55:53 -0400
Subject: [PATCH 5/6] cmd/link/internal/ld: disable
 testWindowsBuildmodeCSharedASLR if CGO isn't enabled

With CGO disabled, the test throws the following error:

ld_test.go:215: build failure: exit status 2
   # command-line-arguments
   loadinternal: cannot find runtime/cgo
---
 src/cmd/link/internal/ld/ld_test.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cmd/link/internal/ld/ld_test.go b/src/cmd/link/internal/ld/ld_test.go
index 3702a4d08f..c741a830d7 100644
--- a/src/cmd/link/internal/ld/ld_test.go
+++ b/src/cmd/link/internal/ld/ld_test.go
@@ -187,6 +187,7 @@ func TestWindowsBuildmodeCSharedASLR(t *testing.T) {
 func testWindowsBuildmodeCSharedASLR(t *testing.T, useASLR bool) {
 	t.Parallel()
 	testenv.MustHaveGoBuild(t)
+	testenv.MustHaveCGO(t)
 
 	dir := t.TempDir()
 
-- 
2.34.1

